### PR TITLE
etcdmain: var log -> plog

### PR DESCRIPTION
**Pull request by yichengq**
So the variable name doesn't mess up with standard package name.

### CODE REVIEW

**Comment by xiang90** [comment]
Can you also clean up the printf (use infof)

**Comment by yichengq** [comment]
@xiang90 sure. added. PTAL

File: etcdmain/etcd.go [hunk 1]
```diff
@@ -61,10 +61,10 @@ func Main() {
 	cfg := NewConfig()
 	err := cfg.Parse(os.Args[1:])
 	if err != nil {
-		log.Printf("error verifying flags, %v. See 'etcd --help'.", err)
+		plog.Errorf("error verifying flags, %v. See 'etcd --help'.", err)
 		switch err {
 		case errUnsetAdvertiseClientURLsFlag:
-			log.Printf("When listening on specific address(es), this etcd process must advertise accessible url(s) to each connected client.")
+			plog.Infof("When listening on specific address(es), this etcd process must advertise accessible url(s) to each connected client.")
```
Comment (xiang90) [inline etcdmain/etcd.go]: plog.Errorf

File: etcdmain/etcd.go [hunk 2]
```diff
@@ -86,19 +86,19 @@ func Main() {
 
 	if cfg.dir == "" {
 		cfg.dir = fmt.Sprintf("%v.etcd", cfg.name)
-		log.Printf("no data-dir provided, using default data-dir ./%s", cfg.dir)
+		plog.Infof("no data-dir provided, using default data-dir ./%s", cfg.dir)
```
Comment (xiang90) [inline etcdmain/etcd.go]: plog.Warningf

File: etcdmain/etcd.go [hunk 3]
```diff
@@ -86,19 +86,19 @@ func Main() {
...
 	}
 
 	which := identifyDataDirOrDie(cfg.dir)
 	if which != dirEmpty {
-		log.Printf("already initialized as %v before, starting as etcd %v...", which, which)
+		plog.Infof("already initialized as %v before, starting as etcd %v...", which, which)
```
Comment (xiang90) [inline etcdmain/etcd.go]: the server is already initialized as ...
and probably noticef?

File: etcdmain/etcd.go [hunk 4]
```diff
@@ -301,22 +301,22 @@ func startProxy(cfg *config) error {
...
 		}
 		err = os.Rename(clusterfile+".bak", clusterfile)
 		if err != nil {
-			log.Printf("proxy: error on updating clusterfile %s", err)
+			plog.Warningf("proxy: error on updating clusterfile %s", err)
 			return clientURLs
 		}
 		if !reflect.DeepEqual(gcls.PeerURLs(), peerURLs) {
-			log.Printf("proxy: updated peer urls in cluster file from %v to %v", peerURLs, gcls.PeerURLs())
+			plog.Warningf("proxy: updated peer urls in cluster file from %v to %v", peerURLs, gcls.PeerURLs())
```
Comment (xiang90) [inline etcdmain/etcd.go]: why this is a warningf? I think this should be a noticef or infof

File: etcdmain/etcd.go [hunk 5]
```diff
@@ -390,12 +390,12 @@ func identifyDataDirOrDie(dir string) dirType {
 		case dirProxy:
 			p = true
 		default:
-			log.Printf("found invalid file/dir %s under data dir %s (Ignore this if you are upgrading etcd)", name, dir)
+			plog.Infof("found invalid file/dir %s under data dir %s (Ignore this if you are upgrading etcd)", name, dir)
```
Comment (xiang90) [inline etcdmain/etcd.go]: warningf

File: etcdmain/etcd.go [hunk 6]
```diff
@@ -415,7 +415,7 @@ func setupLogging(cfg *config) {
 		repoLog := capnslog.MustRepoLogger("github.com/coreos/etcd")
 		settings, err := repoLog.ParseLogLevelConfig(cfg.logPkgLevels)
 		if err != nil {
-			log.Warningf("Couldn't parse log level string: %s, continuing with default levels", err.Error())
+			plog.Warningf("Couldn't parse log level string: %s, continuing with default levels", err.Error())
```
Comment (xiang90) [inline etcdmain/etcd.go]: Could -> couldn't

File: etcdmain/etcd.go (hunk 3 above)
Comment (yichengq) [inline etcdmain/etcd.go]: agreed

File: etcdmain/etcd.go (hunk 4 above)
Comment (yichengq) [inline etcdmain/etcd.go]: hm. will change it to noticef

**Comment by yichengq** [comment]
addressed all comments.

**Comment by xiang90** [comment]
LGTM

**Pull request merged**